oauth2client/contrib/django_util/__init__.py

```diff
@@ -84,9 +84,9 @@
    :caption: urls.py
    :name: urls
 
-   from oauth2client.contrib.django_util.site import urls as oauth2_urls
+   from oauth2client.contrib.django_util import site
 
-   urlpatterns += [url(r'^oauth2/', include(oauth2_urls))]
+   urlpatterns += [url(r'^oauth2/', include(site.oauth2_urls))]
```

Comment:
Since this in an example, it probably makes sense to leave this as it was. @waprin